module.go

```diff
@@ -106,7 +106,8 @@ func getNode(ctx caddy.Context, name string) (*tailscaleNode, error) {
 			Logf: func(format string, args ...any) {
 				app.logger.Sugar().Debugf(format, args...)
 			},
-			Ephemeral: getEphemeral(name, app),
+			Ephemeral:    getEphemeral(name, app),
+			RunWebClient: true,
```

Comment:
I am not familiar with the context here (also not very familiar with the web client). Why would we want to do this for every node?

Reply:
It's generally harmless, since it still relies on the tailnet ACLs to provide access.  However, we can also make it explicitly opt-in.  Added a "webui" config option.

I added this as an `opt.Bool` so that the node-level config only overrides the app-level config if it's explicitly set, which is similar to the behavior of all of the string fields.  I've also updated the node-level `ephemeral` setting to match, and added tests for both.